script/setup

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Setups the repository.
+
+# Stop on errors
+set -e
+
+cd "$(dirname "$0")/.."
+
+python3 -m pip install -r requirements_all.txt
+pre-commit install
```

Comment:
Where is this used?

Reply:
I thought to use pre-commit locally for development, to fix code style problems before pushing.